modules/rosapps/applications/vcdcontroltool/lang/it-IT.rc

```diff
@@ -0,0 +1,69 @@
+/*
+ * PROJECT:     ReactOS Virtual CD Control Tool
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Italian resource file
+ * COPYRIGHT:   Copyright 2019 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+
+LANGUAGE LANG_ITALIAN, SUBLANG_NEUTRAL
+
+IDD_MAINWINDOW DIALOGEX 0, 0, 320, 200
+STYLE DS_SETFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
+EXSTYLE WS_EX_APPWINDOW
+CAPTION "Pannello di controllo dell'unità virtuale CDRom"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    PUSHBUTTON "OK", IDC_MAINOK, 211, 179, 50, 14, WS_CHILD | WS_VISIBLE | WS_TABSTOP
+    CONTROL "", IDC_MAINDEVICES, "SysListView32", LVS_REPORT | LVS_SINGLESEL | LVS_SHOWSELALWAYS | LVS_NOSORTHEADER | WS_CHILD | WS_VISIBLE | WS_BORDER | WS_TABSTOP, 7, 7, 306, 83
+    GROUPBOX "La geometria del driver", IDC_STATIC, 7, 101, 153, 74, BS_GROUPBOX | WS_CHILD | WS_VISIBLE
```

Comment:
Better short to "Geometria del driver"? Because in a phrase is better like this but on an option is better in short (in my opinion)

Reply:
I have thought of such idea way before creating this commit for the PR, but I ended up by adding the article since the words can still fit onto the group-box (it's not an option).

Sure, I can change that.